kong/runloop/balancer.lua

```diff
@@ -270,20 +264,24 @@ do
       healthcheckers[balancer] = hc
 
       balancer.report_http_status = function(ip, port, status)
+        log(DEBUG, "HC ", tostring(hc), " FOR BALANCER ", tostring(balancer), " REPORTING HTTP STATUS ", ip, ":", port, " ", status)
```

Comment:
all these lines containing uppercase DEBUG logs (plus the `print` below) should be removed